frontend/hub/api/utils.tsx

```diff
@@ -81,10 +81,16 @@ export function pulpHrefKeyFn(item: { pulp_href: string }) {
 }
 
 export function collectionKeyFn(item: {
-  collection_version?: { pulp_href: string };
+  collection_version?: { pulp_href: string; version?: string };
   repository?: { name: string };
 }) {
-  return item.collection_version?.pulp_href + '_' + item.repository?.name;
+  return (
+    item.collection_version?.pulp_href +
+    '_' +
+    item.repository?.name +
+    '_' +
+    item.collection_version?.version
```

Comment:
(nothing wrong with this, but .. a situation where the collection version pulp_href is the same but the version string is different should be impossible in the db)